compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -107,6 +107,8 @@ int entry(int argc, char **argv)
              "into one operation and merge reduction indices.");
   add_switch(arser, "--fuse_mul_with_conv",
              "This will fuse Mul operation with a preceding Conv if possible.");
+  add_switch(arser, "--fuse_mul_div",
+             "This will fuse Mul operation with a Div operation if possible.");
```

Comment:
```suggestion
  add_switch(arser, "--fuse_mul_with_div",
             "This will fuse Mul operation with a Div operation whose numerator is const.");
```